JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/api/BackEnd.java

```diff
@@ -72,6 +92,50 @@ public Receive<Event> actor() {
                 .build();
     }
 
+    private Behavior<Event> eventLoginWithKeycloakHandler(final EventLoginWithKeycloakRequest request) {
+        LOGGER.debug("loginWithKeycloak");
+        LOGGER.debug("Logging in {}", request.payload);
+        try {
+            User user2 = PsqlQueries.getUserByEmail("test@yopmail.com");
```

Comment:
Is this test code?

Reply:
Oups forgot to remove that ! Nice catch :)